script/sync-ghes/index.ts

```diff
@@ -45,7 +45,7 @@ async function checkWorkflows(
     });
 
     for (const e of dir) {
-      if (e.isFile()) {
+      if (e.isFile() && extname(e.name) === ".yml") {
```

Comment:
This is because we have a readme in the code-scanning folder right?

Reply:
👍 Yep.